pynucastro/networks/rate_collection.py

```diff
@@ -2035,32 +2035,19 @@ def __repr__(self):
 
 class Explorer:
     """ interactively explore a rate collection """
-    def __init__(self, rc, comp, size=(800, 600),
-                 ydot_cutoff_value=None, rotated=False,
-                 hide_xalpha=False,
-                 always_show_p=False, always_show_alpha=False,
-                 node_size=1000, node_font_size=13):
+    def __init__(self, rc, comp, **kwargs):
         """ take a RateCollection and a composition """
         self.rc = rc
         self.comp = comp
-        self.size = size
-        self.ydot_cutoff_value = ydot_cutoff_value
-        self.always_show_p = always_show_p
-        self.always_show_alpha = always_show_alpha
-        self.hide_xalpha = hide_xalpha
-        self.rotated = rotated
-        self.node_size = node_size
-        self.node_font_size = node_font_size
+        self.kwargs = kwargs
+
+        # we will override any T and rho passed in
+        _ = kwargs.pop("T", None)
+        _ = kwargs.pop("rho", None)
```

Comment:
I don't think assigning to `_` is strictly necessary, but it's fine if you think it looks nicer

Reply:
yes, I guess I can just not assign.  I'll change that.

Reply:
done